adapters/handlers/grpc/v1/prepare_reply_test.go

```diff
@@ -63,6 +63,15 @@ func byteVector(vec []float32) []byte {
 	return vector
 }
 
+func byteVectorMulti(mat [][]float32) []byte {
+	matrix := make([]byte, 2)
+	binary.LittleEndian.PutUint16(matrix, uint16(len(mat[0])))
+	for _, vec := range mat {
+		matrix = append(matrix, byteVector(vec)...)
+	}
+	return matrix
```

Comment:
hmm can you simply use here `byteops.Fp32SliceOfSlicesToBytes` method? or this code does something else than `byteops.Fp32SliceOfSlicesToBytes(`?

Reply:
I think we could but the intent here is to decouple the codepath that is used to create the test requests from the codepath that is used to perform the internal mapping in the backend. If we had the same method for both then a breaking change to `byteops.Fp32SliceOfSlicesToBytes` would not be identified by this test since its effects would be present in the test request too